rmw_opendds_cpp/src/rmw_take.cpp

```diff
@@ -24,8 +24,8 @@
 #include "rmw_opendds_cpp/identifier.hpp"
 
 // include patched generated code from the build folder
-#include "./opendds_static_serialized_dataSupport.h"
-#include "./opendds_static_serialized_data.h"
+#include "opendds_static_serialized_dataTypeSupportC.h"
+// #include "./opendds_static_serialized_data.h"
```

Comment:
This will eventually need to be removed. Unsure as to why it is still there.

Reply:
removed in latest